sdk/core/System.ClientModel/src/Message/PipelineResponse.cs

```diff
@@ -7,34 +7,100 @@
 
 namespace System.ClientModel.Primitives;
 
+/// <summary>
+/// Represents an HTTP response received from a cloud service.
+/// </summary>
 public abstract class PipelineResponse : IDisposable
 {
     // TODO(matell): The .NET Framework team plans to add BinaryData.Empty in dotnet/runtime#49670, and we can use it then.
     internal static readonly BinaryData s_EmptyBinaryData = new(Array.Empty<byte>());
 
     /// <summary>
-    /// Gets the HTTP status code.
+    /// Gets the status code of the HTTP response.
     /// </summary>
     public abstract int Status { get; }
 
     /// <summary>
-    /// Gets the HTTP reason phrase.
+    /// Gets the reason phrase that accompanies the status code on the HTTP
+    /// reponse.
     /// </summary>
     public abstract string ReasonPhrase { get; }
 
+    /// <summary>
+    /// Gets the collection of HTTP request headers.
+    /// </summary>
     public PipelineResponseHeaders Headers => HeadersCore;
 
+    /// <summary>
+    /// Gets or sets the derived-type's value of the response's
+    /// <see cref="Headers"/> collection.
+    /// </summary>
     protected abstract PipelineResponseHeaders HeadersCore { get; }
 
     /// <summary>
-    /// Gets the contents of HTTP response. Returns <c>null</c> for responses without content.
+    /// Gets or sets the contents of the HTTP response.
```

Comment:
This is an odd form.   Did you intent to embed the `<remarks>` in the `<summary>` block?   I think these were meant to be separate blocks.

Reply:
No - thanks for catching!